### PR TITLE
EOS-14269 s3 sanity cleanup issues

### PR DESCRIPTION
**Pull request by abhilekh268**

### CODE REVIEW

File: s3-sanity-test.sh [hunk 1]
```diff
@@ -75,24 +75,46 @@ Operations performed:
...
+   output=$(s3iamcli resetaccountaccesskey -n SanityAccountToDeleteAfterUse --ldapuser sgiamadmin --ldappasswd "$ldappasswd") || echo ""
+   if [[ $output == *"NoSuchEntity"* ]]
+   then
+      echo "Sanity Account doesn't exist"
+      return;
+   else
+      access_key=$(echo -e "$output" | tr ',' '\n' | grep "AccessKeyId" | awk '{print $3}')
+      secret_key=$(echo -e "$output" | tr ',' '\n' | grep "SecretKey" | awk '{print $3}')
+      TEST_CMD="s3cmd --access_key=$access_key --secret_key=$secret_key"
+      $TEST_CMD del "s3://sanitybucket/SanityObjectToDeleteAfterUse" || echo "Failed"
```
Comment (sachitanands) [inline s3-sanity-test.sh:111]: please update echo messages with proper messages like "Failed to delete object "objectName" using s3cmd"

File: s3-sanity-test.sh [hunk 2]
```diff
@@ -140,7 +158,7 @@ TEST_CMD="s3cmd --access_key=$access_key --secret_key=$secret_key"
 
 echo -e "\nCreate bucket - 'sanitybucket': "
 $TEST_CMD mb "s3://sanitybucket"
-
+`
```
Comment (sachitanands) [inline s3-sanity-test.sh:161]: pls remove this quote

File: s3-sanity-test.sh [hunk 3]
```diff
@@ -75,24 +78,46 @@ Operations performed:
...
+
+restore_config() {
+   ls /root/.sgs3iamcli/config.yaml 2> /dev/null || { echo "S3iamcli configuration file is missing" && exit 1; }
+   ls /root/.s3cfg 2> /dev/null || { echo "S3cmd configuration file is missing" && exit 1; }
+   rm -rf /root/.sgs3iamcli
+   sed -i "s/host_base =.*/host_base = s3.seagate.com/g" /root/.s3cfg
+}
+
+cleanup() {
+   update_config
```
[the remark this answers is not in the document]
Comment (abhilekh268) [reply s3-sanity-test.sh:104]: no...it won't be false even cleanup has commands s3cmd and s3iamcli u will need endpoint there without endpoint u can't run unless there is entry in /etc/host.

**Review by bkirunge7 — APPROVED** [review]

**Review by sachitanands — APPROVED** [review]
looks good

**Comment by nileshgovande** [comment]
No Jenkins needed - Merging

**Pull request merged**